src/lib/schematics/update/material/data/method-call-checks.json

```diff
@@ -1,21 +1,4 @@
 [
-  {
```

Comment:
Why did you make this change? was it somehow required for things to compile?

Reply:
I thought it wasn't necessary since the `NativeDateAdapter` exists in `src/cdk/datetime` now. I will put it back.

Reply:
This code is used to automatically migrate people from 5 to 6. We'll eventually add similar transforms to try to automatically update peoples code to fix breaking changes from the datepicker refactor, but for now I think we should just leave this alone